test/e2e/tests/metrics/state-snapshots/errors-after-init-opt-in-background-state.json

```diff
@@ -76,8 +76,6 @@
       "srcTokens": {},
       "srcTopAssets": {},
       "destTokens": {},
-      "destTokensLoadingStatus": "undefined",
```

Comment:
This suggests that these default to `undefined`. Is that intended?

Reply:
Good, these are optional: https://github.com/MetaMask/metamask-extension/pull/28373/files#diff-5f37a2a6ea0903094f4dbc84f86b96733525a125a5c66770527bd7e32a5b832aR40

This seems OK then

Reply:
It is! This is the state when the list is unfetched. These are going to be deprecated by the next release in favor of the multichain asset lists so this will be cleaned up soon